ftl/python/layer_builder.py

```diff
@@ -214,7 +224,7 @@ def __init__(self,
                  pip_cmd=None,
                  venv_cmd=None,
                  cache=None):
-        super(RequirementsLayerBuilder, self).__init__()
+        super(PipfileLayerBuilder, self).__init__()
```

Comment:
did you mean to change this? was this a bug?

Reply:
yes, this should be PipfileLayerBuilder